sparkbox/apprentices-page-next
# feat: callToAction

## Description

- created HTML
- styled the layout for just mobile view
- added all images to public folder
- styled tablet view
- styled desktop view

FSA21V2-146

### Spec

Designs: [link to design](https://www.figma.com/file/eeSQQIRz0R5Ag9SIJJH6Hx/apprentices.sparkbox.com?node-id=104%3A6929)

See Story: [FSA21V2-146](https://sparkbox.atlassian.net/browse/FSA21V2-146)

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci`
4. Confirm linter passes: `npm run lint`

## Validation

The following has been completed by the developer:

_[delete anything irrelevant to this PR]_


### Browser Testing

_[delete if irrelevant to this issue]_

#### Gold Level Browsers

In these browsers, all content is accessible and design matches exactly. In most cases, bugs should be resolved before merge.

**macOS**


**Windows**


**Mobile**


#### Silver Level Browsers

In these browsers, all content is readable, but design may be inaccurate.

**Windows**


**Mobile**


#### Bronze Level Browsers

In these browsers, our only goal is content readability.

**Windows**
